tests/Knp/Menu/Tests/Iterator/IteratorTest.php

```diff
@@ -19,15 +19,7 @@ public function testIterator()
 
     public function testRecursiveIterator()
     {
-        // Adding an item which does not provide a RecursiveIterator to be sure it works properly.
```

Comment:
With the sort on addChild, this test become impossible because prophecy use exceptions and php don't support it while sorting (it throw a warning and make tests fail).

https://bugs.php.net/bug.php?id=50688